streaming/src/main/scala/org/apache/spark/streaming/api/java/JavaStreamingContext.scala

```diff
@@ -279,13 +289,14 @@ class JavaStreamingContext(val ssc: StreamingContext) extends Closeable {
    */
   def fileStream[K, V, F <: NewInputFormat[K, V]](
       directory: String,
+      depth: Int = 1,
```

Comment:
This breaks the Java API since there is no longer a version without a depth param in Java right?